translations/ai_rules.tex/cs.po

```diff
@@ -48,6 +48,8 @@ msgid ""
 "{2}\n"
 "\n"
 msgstr ""
+"{2}\n"
```

Comment:
nit: it's not necessary to copy msgid into msgstr if there's nothing to translate. But it probably doesn't hurt either...

Reply:
Hi, this was actually autogenerated by `po4a po4a.cfg`